bvm/ballerina-core/src/main/java/org/ballerinalang/util/exceptions/SemanticErrors.java

```diff
@@ -132,7 +132,10 @@
     UNKNOWN_FIELD_IN_JSON_STRUCT("unknown.field.in.json.struct", "SEMANTIC_0108"),
     RETRY_STMT_NOT_ALLOWED_HERE("retry.stmt.not.allowed.here", "SEMANTIC_0109"),
     INVALID_RETRY_COUNT("invalid.retry.count", "SEMANTIC_0110"),
-    ARGUMENTS_COUNT_MISMATCH("arguments.count.mismatch", "SEMANTIC_0109")
+    ARGUMENTS_COUNT_MISMATCH("arguments.count.mismatch", "SEMANTIC_0109"),
+    NO_SUCH_DOCUMENTABLE_ATTRIBUTE("no.such.documentable.attribute", "SEMANTIC_0110"),
```

Comment:
These properties are no longer used for semantic error. Please remove them.

Reply:
+1